src/test/ui/abi/abi-sysv64-arg-passing.rs

```diff
@@ -28,36 +28,40 @@
 // ignore-arm
 // ignore-aarch64
 // ignore-windows
+// ignore-uefi rust_test_helpers do not work
 
 // note: windows is ignored as rust_test_helpers does not have the sysv64 abi on windows
 
 #[allow(dead_code)]
 #[allow(improper_ctypes)]
-
 #[cfg(target_arch = "x86_64")]
 mod tests {
     #[repr(C)]
     #[derive(Copy, Clone, PartialEq, Debug)]
     pub struct TwoU8s {
-        one: u8, two: u8
+        one: u8,
+        two: u8,
```

Comment:
There is lots of reformatting done to test-files. This is a lot of noise in this PR. Is this required because rustfmt only runs on files where changes are made? It is really hard to find the changes that matter.

Reply:
I can try to fix that, but some formatting changes are done by the git hooks that rust installs, so I'm not sure how to deal with that.

Reply:
Removed almost all of the formatting.